src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -438,29 +438,126 @@ public void OnWindowUpdate(int amount)
                 }
             }
 
+            private const int FirstHPackRequestPseudoHeaderId = 1;
+            private const int LastHPackRequestPseudoHeaderId = 7;
+            private const int FirstHPackStatusPseudoHeaderId = 8;
+            private const int LastHPackStatusPseudoHeaderId = 14;
+            private const int FirstHPackNormalHeaderId = 15;
+            private const int LastHPackNormalHeaderId = 61;
+
+            private static readonly int[] HPackStaticStatusCodeTable = new int[LastHPackStatusPseudoHeaderId - FirstHPackStatusPseudoHeaderId + 1] { 200, 204, 206, 304, 400, 404, 500 };
+
+            private static readonly (HeaderDescriptor descriptor, ReadOnlyMemory<byte> value)[] HPackStaticHeaderTable = new (HeaderDescriptor, ReadOnlyMemory<byte>)[LastHPackNormalHeaderId - FirstHPackNormalHeaderId + 1]
```

Comment:
Why store `ReadOnlyMemory<byte>` rather than `byte[]`? The latter is smaller to store, and it should be faster to create a span from.

Reply:
No particular reason. Will change.